apps/src/applab/VisualizationOverlay.js

```diff
@@ -121,7 +124,16 @@ VisualizationOverlay.prototype.onSvgMouseMove_ = function (event) {
 
   this.mousePos_.x = event.clientX;
   this.mousePos_.y = event.clientY;
-  this.mousePos_ = this.mousePos_.matrixTransform(this.screenSpaceToAppSpaceTransform_);
+  var draggingElement = $(".ui-draggable-dragging");
```

Comment:
Seems a little unfortunate to have VisualizationOverlay need to know about ui-draggable-dragging. I could pass in as a prop, but not sure that's much better (we could make the selector a prop, but we would still need to look for dragging elements in real time). Thoughts?